lcc_lokavaluto_app_connection/models/credit_request.py

```diff
@@ -58,7 +58,7 @@ def write(self, vals):
         for request in self:
             if request.state == "pending":
                 # The top up has been paid, the credit process can start
-                if request.partner_id.company_id.activate_automatic_topup:
+                if self.env.user.company_id.activate_automatic_topup:
```

Comment:
are we sure to want to check who is actively calling this `write` ? Not real sure to understand why this change.